virt/release_notes/virt-4-18-release-notes.adoc

```diff
@@ -125,6 +125,9 @@ Some features in this release are currently in Technology Preview. These experim
 
 link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope]
 
+//CNV-52362: IBM Z
+* You can use {VirtProductName} in a logical partition (LPAR) on IBM{nbsp}Z® and IBM®{nbsp}LinuxONE (s390x architecture) systems.
```

Comment:
```suggestion
* You can use {VirtProductName} in a logical partition (LPAR) on {ibm-z-name} and {ibm-linuxone-name} (s390x architecture) systems.
```

Reply:
Great comment, thanks! I was planning to add the common attributes in a different PR and didn't realize they already existed. I will use them here.